packages/stream/solana/utils.ts

```diff
@@ -7,26 +7,37 @@ import {
   isSignerWallet,
   ThrottleParams,
 } from "@streamflow/common/solana";
-import BN from "bn.js";
+import BigNumber from "bignumber.js";
 
 import { streamLayout } from "./layout.js";
 import { DecodedStream, BatchItem, BatchItemResult } from "./types.js";
 import { SOLANA_ERROR_MAP, SOLANA_ERROR_MATCH_REGEX } from "./constants.js";
 
 const decoder = new TextDecoder("utf-8");
-const LE = "le"; //little endian
+
+const toUInt64String = (uintArr: Uint8Array): string =>
```

Comment:
please describe in comments or another way when and why it is necessary to use these functions

Reply:
BigNumber doesn't have native support for little endian parsing of a byteArray so this is essentially that - will add a comment like this